config/karma.config.js

```diff
@@ -3,11 +3,12 @@
 
 import webpackConfig from './webpack.config';
 
-var coverageReporters = [{
+const coverageReporters = [{
   type: 'text-summary',
 }];
 
-var newWebpackConfig = Object.assign({}, webpackConfig, {
+// eslint-disable-next-line prefer-const
+let newWebpackConfig = Object.assign({}, webpackConfig, {
```

Comment:
My concern with the signalling intent thing is that it creates a disconnect between what you think something means and what it actually allows. It's easy to say "this is going to be a constant, don't change it" but once you pass it to a function or it gets a few (dozen? hundred?) lines further down in the code all bets are off as to whether that intent will be upheld.

Realistically there's no real benefit to using `const` here so I'd be fine with using `let` and never using `const` unless something is actually immutable. So then if we use `const` it should be with something immutable like `42` or `'foo'` or `Immutable.Map({foo, bar})`.